### DISCUSSION TITLE
Offline new console creation

### DISCUSSION THREAD

**Discussion by hmrguez** [dotnet/csharplang#8504] [Language Ideas]
From .NET forward the new console template now requires dotnet restore because there are new nuggets in the template. This makes it inconvenient for computing at the edge for testing some features and overall starting with the language in low internet access areas. Can you regress back to what it used to be or offer some kind of feature for those required nuggets to be included in the .net sdk and the console project wouldn't need the dotnet restore command?

**Comment by HaloFour** [comment]
That sounds like a [dotnet SDK](https://github.com/dotnet/sdk) question?  The language itself is agnostic to all of that.
